src/components/Home/Index.jsx

```diff
@@ -198,6 +205,103 @@ function Index() {
           </Col>
         </Row>
       </Section>
+
+      <Section title="About Us" id="about">
```

Comment:
Is it really important to change the ID here, or should we go with the one that is automatically generated from the title?

Manually setting it can help us with stable links but is also more code to maintain...

Reply:
totally fine to go with the auto-generated ID, I just forgot about it.